### PR TITLE
fix horrible snaxi vendor placement, fix punctuation in same vending machine + other places

### PR DESCRIPTION
**Pull request by nervere**
## What this does
Fixes this atrocity on Snaxi, as well as a bunch of punctuation issues that have really been bugging me.
qga8eniFTh

After:
image

Vendor moved to this room just below it (rec room):
image

## Changelog

:cl:
 * tweak: Fixed some poor vendor placement on Snaxi.

### CODE REVIEW

**Comment by SonixApache** [comment]
ah so this IS mapped
i was yelling at the gods for a bit the first time i saw it too

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -4117,11 +4117,11 @@ var/global/list/obj/item/weapon/paper/lotto_numbers/lotto_papers = list()
 
 /obj/machinery/vending/art
 	name = "\improper Le Patron des Arts"  //Mécène won't render
-	desc = "A vendor for art supplies. Pas sans mon béret !"
+	desc = "A vendor for art supplies. Pas sans mon béret!"
```
Comment (Kurfursten) [inline code/game/machinery/vending.dm]: This is correct as-is, in French a space is inserted before a two-part punctuation

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -4117,11 +4117,11 @@ var/global/list/obj/item/weapon/paper/lotto_numbers/lotto_papers = list()
...
 	product_slogans = list(
 		"Il est temps pour vous de jeter des perles devant ces porcs incultes.",
 		"Oui, j'ai capitulé. Vous pouvez aussi acheter des crayons maintenant.",
-		"Il y a la peinture sur toile, et puis il y a l'art véritable. Pouvez-vous voir la différence ?"
+		"Il y a la peinture sur toile, et puis il y a l'art véritable. Pouvez-vous voir la différence?"
```
Comment (Kurfursten) [inline code/game/machinery/vending.dm]: This is correct as-is, in French a space is inserted before a two-part punctuation

**Comment by Kurfursten** [comment]
Very weird to bundle this with grammar fixes. Also, mapdiffbot isn't working so you'll need an after screenshot.

File: code/game/machinery/vending.dm (hunk 2 above)
Comment (nervere) [reply code/game/machinery/vending.dm]: Fixed.

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (nervere) [reply code/game/machinery/vending.dm]: Fixed.

**Comment by nervere** [comment]
> Very weird to bundle this with grammar fixes. Also, mapdiffbot isn't working so you'll need an after screenshot.

Before vs after added to OP. The horrible French punctuation on the vendor is what made me just do a grep search and change it. Honestly even if it's correct in French it looks dumb in our English-language game. I won't argue though because this is unatomic for no reason other than that I was being lazy about making a new branch and re-doing my work.

**Pull request merged**